requirements/debug

```diff
@@ -0,0 +1,5 @@
+# These are dependencies to help with debug and profiling, so we always want
+# the latest versions.
+
+# Interactive console on steroids (even makes your coffee)!
+ipython
```

Comment:
What is this useful for? Is it a suggestion to contributors, or is it actually used by OpenFisca-Core anywhere?